src/views/assets/Cloud/Strategy/StrategyList.vue

```diff
@@ -50,7 +58,11 @@ export default {
         hasImport: false,
         hasExport: false,
         hasMoreActions: false,
-        createRoute: 'CloudStrategyCreate'
+        createRoute: () => {
+          return {
+            name: 'CloudStrategyCreate', query: { category: this.$route.query.category || 'host' }
+          }
+        }
       }
     }
   },
```

Comment:
I would be able to analyze the provided code more effectively with more specific information about its structure and contents. The context and specifications you have not included make it impossible for me to properly evaluate or provide advice on potential irregularities, issues, optimization suggestions, etc.